src/Tests/Interop/Interop.idl

```diff
@@ -0,0 +1,154 @@
+// Modern IDL 3.0: https://docs.microsoft.com/en-us/uwp/midl-3/intro
+
+namespace Windows.Interop
```

Comment:
These interop interfaces are already defined in winmd format by the [win32metadata](https://github.com/microsoft/win32metadata/) project. You can find the [nuget package with the winmd over here](https://www.nuget.org/packages/Microsoft.Windows.SDK.Win32Metadata/). The C#/WinRT project could simply be updated to parse this additional winmd file and generate these definitions/wrappers automatically as the [Windows crate for Rust](https://github.com/microsoft/windows-rs/) already does. The whole point of language projections is to get out of the business of writing such language bindings by hand. :wink:

Note also that the approach used here of using MIDL3 will only work for those interop interfaces that derive from IInspectable. The win32 metadata supports everything. @sotteson1 @mikebattista

Reply:
@angelazhangmsft, sigh, this is my oversight.  I meant to follow up on this as a way to model the interops.  Kenny is right - we should absolutely use the Microsoft.Windows.SDK.Win32Metadata package, with appropriate CsWinRTFilters settings, to project the interops we can (and model the remainder in C#, if necessary).

Reply:
Note that ILDASM produces IL rather than IDL. [ILSpy](https://github.com/icsharpcode/ILSpy/releases) is also rather handy. 

![image](https://user-images.githubusercontent.com/9845234/116583000-bfd1f400-a8ca-11eb-96b2-80a8eadd2466.png)

Reply:
> how does win32metadata handle IUnknown-based interops

The `IAccountsSettingsPaneInterop` example above is an interop interface that derives from `IInspectable`, but win32metadata can also define non-WinRT interfaces. Here's good old `IMemoryBufferByteAccess`, which derives directly from `IUnknown`.

![image](https://user-images.githubusercontent.com/9845234/116583618-64543600-a8cb-11eb-94ae-2bd336c8ae5e.png)

Reply:
Yes, IL (not IDL).  ILSpy is terrific, but I'm seeing exceptions trying to display definitions for interfaces (like IMemoryBufferByteAccess):
  **System.ArgumentException: An item with the same key has already been added.)**

so suggested ildasm as a search technique.

Reply:
@scottj1s are you using version 7? https://github.com/icsharpcode/ILSpy/releases

Reply:
I am now ;-)  yes, problem solved

Reply:
Upon investigation, decided on defining the APIs in IDL instead of using win32metadata, as their metadata isn't "windows runtime" metadata